ocs_ci/deployment/vmware.py

```diff
@@ -198,6 +199,19 @@ def deploy_prereq(self):
                     config.ENV_DATA.get('memory')
                 )
 
+            # increase CPUs
+            worker_num_cpus = config.ENV_DATA.get('worker_num_cpus')
+            master_num_cpus = config.ENV_DATA.get('master_num_cpus')
+            if worker_num_cpus or master_num_cpus:
+                with open(constants.VSPHERE_MAIN, 'r') as fd:
+                    obj = hcl.load(fd)
+                    if worker_num_cpus:
+                        obj['module']['compute']['num_cpu'] = worker_num_cpus
+                    if master_num_cpus:
+                        obj['module']['control_plane']['num_cpu'] = master_num_cpus
+                dump_data_to_json(obj, f"{constants.VSPHERE_MAIN}.json")
+                os.rename(constants.VSPHERE_MAIN, f"{constants.VSPHERE_MAIN}.backup")
```

Comment:
If I understand it well, the newly generated config is saved as *JSON*, because the `hcl` module doesn't support dump of the data in the *HCL* format. Is my assumption correct?

Reply:
@vavuthu thanks for the confirmation, would it make sense to add this note to short comment in the code?

Reply:
Done